release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -98,6 +98,11 @@ In {product-title} 4.7, _Cluster Logging_ became _Red Hat OpenShift Logging_. Fo
 [id="ocp-4-10-scale"]
 === Scale
 
+
+[id="ocp-4-10-hyper-threading"]
+==== Hyper-threading aware CPU manager policy (Technology Preview)
+Hyper-threading aware CPU manager policy in Openshift Container Platform is now enabled by default. Hyper-threads are abstracted by the hardware as logical processors. Hyper-threading allows a single physical processor to execute two heavyweight threads (processes) at the same time, dynamically sharing processor resources.
+
```

Comment:
seems correct. The u/s docs should hopefully explain the scope and the implications of this feature: https://kubernetes.io/docs/tasks/administer-cluster/cpu-management-policies/#static-policy-options